src/components/inputs/TextInput.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from 'react';
+import { Text, TextStyle, View, ViewStyle } from 'react-native';
+import { TextInput as BaseTextInput, TextInputProps } from 'react-native-paper';
+import { useTheme } from '../../styles/themes';
+
+interface Props extends TextInputProps {
+    placeHolder: string;
+    style?: ViewStyle;
+    onChangeText?: (value: string) => void;
+    value: string;
+    isError?: boolean;
+    renderError?: () => any;
+    disabled?: boolean;
+    label: string;
+    informationText: string;
+}
+
+export const TextInput = ({
+    placeHolder,
+    style = {},
+    value,
+    onChangeText,
+    isError,
+    renderError,
+    disabled,
+    label,
+    informationText,
+    ...props
+}: Props) => {
+    const theme = useTheme();
+
+    const [focused, setFocused] = useState<boolean>(false);
+
+    const containerStyle: ViewStyle = {
+        marginBottom: theme.sw.spacing.l,
+        ...style,
+    };
+
+    const _inputSyle: TextStyle = {
```

Comment:
sometimes you have '_' as prefix sometimes you don't. We do not have any convention on that yet but I think we prefer to not use '_'.